release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2127,6 +2127,28 @@ This behavior has been fixed, and the references are now accepted and correctly
 [id="ocp-4-14-olm-bug-fixes"]
 ==== Operator Lifecycle Manager (OLM)
 
+* Before {product-title} 4.14.2, Operator Lifecycle Manager (OLM) could overwrite privileged cluster role bindings used by {product-title} and Kubernetes resources if you created an Operator group with the same name. In {product-title} 4.14.2 and later, if you create an Operator group that conflicts with a privileged cluster role binding, OLM renames the cluster role binding and adds a hash value avoid conflicts. This renaming only occurs if you Operator group conflicts with a privileged cluster role binding. You must ensure that the name of your Operator group does not conflict with other cluster role bindings that exist on the cluster.
++
+.Renaming syntax
+[source,text]
+----
+olm.og.<operator_group_name>.<admin_edit_or_view>-<hash_value>
+----
++
+OLM treats the following cluster role bindings as privileged:
```

Comment:
Same here, should be `cluster roles`, not `cluster role bindings`.

Reply:
@jianzhangbjz Updates applied. Thank you!